src/tasks.ts

```diff
@@ -44,7 +44,16 @@ async function provideJuliaTasksForFolder(folder: vscode.WorkspaceFolder): Promi
             let testTask = new vscode.Task({ type: 'julia', command: 'test' }, folder, `Run tests`, 'julia', new vscode.ProcessExecution(g_settings.juliaExePath, ['--color=yes', '-e', 'Pkg.test(Base.ARGS[1])', rootPath]), "");
             testTask.group = vscode.TaskGroup.Test;
             testTask.presentationOptions = { echo: false };
-            result.push(testTask);
+            result.push(testTask);   
+        }
+
+        if (await fs.exists(path.join(rootPath, 'REQUIRE'))) {
```

Comment:
That is probably a bit too lose, someone might have opened just the ``test`` folder of a package, and that can also contain a ``REQUIRE`` file. I think a more robust version would be to simply test whether the folder is a child folder of ``.julia/v0.x``, right? [This](https://github.com/JuliaEditorSupport/julia-vscode/blob/master/src/openpackagedirectory.ts#L19) line would give you the package root folder, and then we can just check whether the current folder is a direct (!) child of that, right?